src/System.Drawing.Common/src/System/Drawing/Graphics.cs

```diff
@@ -562,5 +583,1930 @@ public void RotateTransform(float angle, MatrixOrder order)
+                count,
+                gpRegions));
+
+            return regions;
+        }
+
+        /// <summary>
+        /// Draws the specified image at the specified location.
+        /// </summary>
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly")]
+        public void DrawImage(Image image, PointF point)
+        {
+            DrawImage(image, point.X, point.Y);
+        }
+
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly")]
+        public void DrawImage(Image image, float x, float y)
+        {
+            if (image == null)
+                throw new ArgumentNullException(nameof(image));
+
+            int status = Gdip.GdipDrawImage(
+                new HandleRef(this, NativeGraphics), new HandleRef(image, image.nativeImage),
+                x, y);
+
+            IgnoreMetafileErrors(image, ref status);
+            CheckErrorStatus(status);
+        }
+
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly")]
+        public void DrawImage(Image image, RectangleF rect)
+        {
+            DrawImage(image, rect.X, rect.Y, rect.Width, rect.Height);
+        }
+
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly")]
+        public void DrawImage(Image image, float x, float y, float width, float height)
+        {
+            if (image == null)
+                throw new ArgumentNullException(nameof(image));
+
+            int status = Gdip.GdipDrawImageRect(
+                new HandleRef(this, NativeGraphics),
+                new HandleRef(image, image.nativeImage),
+                x, y,
+                width, height);
+
+            IgnoreMetafileErrors(image, ref status);
+            CheckErrorStatus(status);
+        }
+
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly")]
+        public void DrawImage(Image image, Point point)
+        {
+            DrawImage(image, point.X, point.Y);
+        }
+
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly")]
+        public void DrawImage(Image image, int x, int y)
+        {
+            if (image == null)
+                throw new ArgumentNullException(nameof(image));
+
+            int status = Gdip.GdipDrawImageI(
+                new HandleRef(this, NativeGraphics),
+                new HandleRef(image, image.nativeImage),
+                x, y);
+
+            IgnoreMetafileErrors(image, ref status);
+            CheckErrorStatus(status);
+        }
+
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly")]
+        public void DrawImage(Image image, Rectangle rect)
+        {
+            DrawImage(image, rect.X, rect.Y, rect.Width, rect.Height);
+        }
+
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly")]
```

Comment:
Do all these `SuppressMessage` still apply?

Reply:
I guess not, I don't think FxCop is enabled for corefx, correct?

A quick search shows 149 `SuppressMessage` attributes across System.Drawing. Would it be OK to remove them all in a separate PR?

Reply:
FxCop is now enabled for corefx, that is why I wonder if we can remove these and see what happens.

But yeah, a separate PR would be great. If you have the time, it would be appreciated.